docker/Dockerfile

```diff
@@ -101,12 +101,17 @@ RUN git clone -b ${TORCH_VISION_BRANCH} https://github.com/pytorch/vision.git &&
     pip install dist/*.whl && cd ..
 ARG TORCH_BENCH_BRANCH=main
 ARG TORCH_BENCH_COMMIT=default
+ARG TRANSFORMERS_VERSION=main
 ARG HF_HUB_TOKEN=hf_xxx
 ENV HUGGING_FACE_HUB_TOKEN ${HF_HUB_TOKEN}
 RUN git clone -b ${TORCH_BENCH_BRANCH} https://github.com/pytorch/benchmark.git && \
     cd benchmark && if [ "${TORCH_BENCH_COMMIT}" = "default" ]; then git checkout `cat /workspace/pytorch/.github/ci_commit_pins/torchbench.txt`; else git checkout ${TORCH_BENCH_COMMIT}; fi && pip install --no-deps -r requirements.txt && \
     pip install --no-cache Jinja2==3.1.2 markupsafe==2.0.1 beartype==0.15.0 mpmath==1.3.0 && \
-    python install.py --continue_on_fail && \
+    python install.py --continue_on_fail && cd .. && \
+    git clone -b ${TRANSFORMERS_VERSION} https://github.com/huggingface/transformers.git && \
+    cd transformers && git checkout `cat /workspace/pytorch/.ci/docker/ci_commit_pins/huggingface.txt` && git submodule sync && git submodule update --init --recursive && \
+    python setup.py bdist_wheel && \
+    pip install dist/*.whl && cd .. && \
```

Comment:
add `--force-reinstall` ?

Reply:
No need to --force-reinstall when ```pip install dist/*.whl```. The old version will be uninstalled.

![image](https://github.com/chuanqi129/inductor-tools/assets/84730719/fcc7031d-e534-4bc0-8369-53ed1090a8d5)